### PR TITLE
Adding documentation for Serf telemetry metrics

### PR DESCRIPTION
**Pull request by tryan225**
Updating docs/agent/telemetry reference to include serf member events. Fixes https://github.com/hashicorp/consul/issues/5400

### CODE REVIEW

File: website/source/docs/agent/telemetry.html.md [hunk 1]
```diff
@@ -909,12 +909,30 @@ These metrics give insight into the health of the cluster as a whole.
...
+  </tr>
   <tr>
     <td>`consul.serf.member.flap`</td>
     <td>Available in Consul 0.7 and later, this increments when an agent is marked dead and then recovers within a short time period. This can be an indicator of overloaded agents, network problems, or configuration errors where agents can not connect to each other on the [required ports](/docs/agent/options.html#ports).</td>
     <td>flaps / interval</td>
     <td>counter</td>
   </tr>
+  <tr>
+    <td>`consul.serf.member.join`</td>
+    <td>This increments when an agent joins the cluster.</td>
```
Comment (freddygv) [inline website/source/docs/agent/telemetry.html.md]: This counter is also incremented when a member re-joins. May be helpful to call that out.

Something like:
> This increments when an agent joins the cluster. If an agent flapped or failed this counter is also incremented when it rejoins.

File: website/source/docs/agent/telemetry.html.md [hunk 2]
```diff
@@ -909,12 +909,30 @@ These metrics give insight into the health of the cluster as a whole.
...
+  </tr>
   <tr>
     <td>`consul.serf.member.flap`</td>
     <td>Available in Consul 0.7 and later, this increments when an agent is marked dead and then recovers within a short time period. This can be an indicator of overloaded agents, network problems, or configuration errors where agents can not connect to each other on the [required ports](/docs/agent/options.html#ports).</td>
     <td>flaps / interval</td>
     <td>counter</td>
   </tr>
+  <tr>
+    <td>`consul.serf.member.join`</td>
+    <td>This increments when an agent joins the cluster. If an agent flapped or failed this counter is also incremented when it re-joins.</td>
```
Comment (judithpatudith) [inline website/source/docs/agent/telemetry.html.md]:
```suggestion
    <td>This increments when an agent joins the cluster. If an agent flapped or failed this counter also increments when it re-joins.</td>
```

Comment (tryan225) [reply website/source/docs/agent/telemetry.html.md]: Good catch!

**Comment by freddygv** [comment]
Thanks @tryan225 !

**Pull request merged**